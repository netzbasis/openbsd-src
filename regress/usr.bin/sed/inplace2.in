#	$OpenBSD: inplace2.in,v 1.1 2015/07/17 20:42:03 jasper Exp $

PROG=	sed
SRCS=	compile.c main.c misc.c process.c

.include <bsd.prog.mk>
