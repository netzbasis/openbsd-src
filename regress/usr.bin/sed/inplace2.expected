#	$OpenBSD: inplace2.expected,v 1.1 2015/07/17 20:42:03 jasper Exp $

GROP=	sed
SRCS=	compile.c main.c misc.c process.c

.include <bsd.prog.mk>
